File: Java/Gate Two/TaskTen.java
import java.util.Scanner;
public class TaskTen{

	public static void main (String [] args){
	
		Scanner input = new Scanner (System.in);

		System.out.println("Enter a number: ");
		int number = input.nextInt();

		int newNumber = number;
		int counter = 0;

		while (newNumber != 0){
			newNumber /= 10;
			counter++;
		}
		System.out.printf("%d has %d digits%n", number, counter);
	}

}

File: JavaScript/Gate Two/TaskTen.js
const prompt = require('prompt-sync')();

let number = Number(prompt("Enter a number: "));

let newNumber = number;
let count = 0;

while(newNumber != 0){

	newNumber = Math.floor(newNumber / 10);
	count++;

}

console.log("%d has %d digits", number, count);